examples/rbd/pod-ephemeral.yaml

```diff
@@ -0,0 +1,22 @@
+---
+kind: Pod
+apiVersion: v1
+metadata:
+  name: csi-rbd-demo-ephemeral-pod
+spec:
+  containers:
+    - name: web-server
+      image: docker.io/library/nginx:latest
+      volumeMounts:
+        - mountPath: /myspace
+          name: mypvc
+  volumes:
+    - name: mypvc
+      ephemeral:
+        volumeClaimTemplate:
+          spec:
+            accessModes: [ReadWriteOnce]
```

Comment:
The other examples use the cleaner (IMHO) format of

```yaml
accessModes:
  - ReadWriteOnce
```

it is probably better to use the same formatting in all examples we have